### PR TITLE
[Cortex XDR] Fix Error Message

### PR DESCRIPTION
**Pull request by shmuel44**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link ](https://jira-hq.paloaltonetworks.local/browse/CIAC-7434)to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

### CODE REVIEW

File: Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py [hunk 1]
```diff
@@ -3255,7 +3256,7 @@ def get_original_alerts_command(client: CoreClient, args: Dict) -> CommandResult
     reply = copy.deepcopy(raw_response)
     alerts = reply.get('alerts', [])
     filtered_alerts = []
-    for i, alert in enumerate(alerts):
+    for _i, alert in enumerate(alerts):
```
Comment (dorschw) [inline Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py:3259]: no need for enumerate here if we don't use `_i`, just `for alert in alerts` would do

File: Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py [hunk 2]
```diff
@@ -3409,7 +3410,7 @@ def get_dynamic_analysis_command(client: CoreClient, args: Dict) -> CommandResul
...
```
Comment (dorschw) [inline Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py:3413]: same here

File: Packs/Core/ReleaseNotes/2_0_1.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+
+##### Investigation & Response
+- Fix an issue where an error message did not appear.
```
Comment (dorschw) [inline Packs/Core/ReleaseNotes/2_0_1.md:5]:
```suggestion
- Fix an issue where the integration failed when searching for a user that did not exist.
```

File: Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule_test.py [hunk 4]
```diff
@@ -138,7 +138,7 @@ def test_endpoint_command(requests_mock):
                                         'IsIsolated': 'No'}]}
 
     results = outputs[0].to_context()
-    for key, val in results.get("EntryContext").items():
+    for key, _val in results.get("EntryContext").items():
         assert results.get("EntryContext")[key] == get_endpoints_response[key]
```
Comment (dorschw) [inline Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule_test.py:142]:
```suggestion
    for key, value in results.get("EntryContext", {}).items():
        assert get_endpoints_response[key] == value
```

File: Packs/Core/ReleaseNotes/2_0_2.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Investigation & Response
+
+- Fixed an issue where the integration failed when searching for a user, host, group, or role that did not exist in the following commands:
```
Comment (dorschw) [inline Packs/Core/ReleaseNotes/2_0_2.md:6]:
```suggestion
- Fixed an issue where the integration failed when searching for a value that does exist, when running one of the following commands:
```

**Review by dorschw — APPROVED** [review]

File: Packs/Core/ReleaseNotes/2_0_2.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Investigation & Response
+
+- Fixed an issue where the integration failed when searching for a value that does exist, when running one of the following commands:
+    - ***core-list-risky-users***
+    - ***core-list-risky-hosts***
+    - ***core-list-user-groups***
+    - ***core-list-roles***
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/2_0_2.md:10]:
```suggestion
Fixed an issue where the integration failed when searching for a value exists, when running one of the following commands:
- ***core-list-risky-users***
- ***core-list-risky-hosts***
- ***core-list-user-groups***
- ***core-list-roles***
```

File: Packs/CortexXDR/ReleaseNotes/5_0_4.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Fixed an issue where the integration failed when searching for a value that does exist, when running one of the following commands:
+    - ***xdr-list-risky-users***
+    - ***xdr-list-risky-hosts***
+    - ***xdr-list-user-groups***
+    - ***xdr-list-roles***
```
Comment (ShirleyDenkberg) [inline Packs/CortexXDR/ReleaseNotes/5_0_4.md:10]:
```suggestion
Fixed an issue where the integration failed when searching for a value exists, when running one of the following commands:
- ***xdr-list-risky-users***
- ***xdr-list-risky-hosts***
- ***xdr-list-user-groups***
- ***xdr-list-roles***
```

**Comment by ShirleyDenkberg** [comment]
@dorschw Doc review completed.

File: Packs/Core/ReleaseNotes/2_0_2.md (hunk 6 above)
Comment (shmuel44) [reply Packs/Core/ReleaseNotes/2_0_2.md:10]: this is typo issue, i mean *doesn't*  exist

**Pull request merged**